File: Java/abc016A.java
import java.util.Scanner;

class Main{
  public static void main(String args[]){
    Scanner scan = new Scanner(System.in);
    int n = scan.nextInt();
    int m = scan.nextInt();
    int s = n % m;
    if(s == 0){
      System.out.println("YES");
    }else{
      System.out.println("NO");
    }
  }
}

File: C++/abc016A.cpp
#include <iostream>
#include <string>
using namespace std;

int main()
{
  int n,m,s;
  cin >> n;
  cin >> m;
  s = n % m;
  if( s == 0){
    cout << "YES" << endl;
  }else{
    cout << "NO" << endl;
  }
}